articles/azure-monitor/insights/container-insights-faq.md

```diff
@@ -9,6 +9,46 @@ ms.date: 10/15/2019
 
 This Microsoft FAQ is a list of commonly asked questions about Azure Monitor for containers. If you have any additional questions about the solution, go to the [discussion forum](https://feedback.azure.com/forums/34192--general-feedback) and post your questions. When a question is frequently asked, we add it to this article so that it can be found quickly and easily.
 
+## I dont see 'Image' and 'Name' fields populated for my logs in ContainerLog table
+
+Starting with the agent version ciprod12042019 (and later) , by default, these two fields are not populated for every logline, to save customer costs on log data collected. There are two options to get these field values -
+
+- option-1 
+
+Join other tables to get these field values:
+
+Please alter your queries to get 'Image' and 'ImageTag' fields from 'ContainerInventory' table by joining on 'ContainerID' field. You can get the 'Name' field (as it used to appear in ContainerLog table) from 'KubepodInventory' table's 'ContainerName' field by joining on 'ContainerID' field. This is the recommended option.

```

Comment:
Reword to - Modify your queries to include *Image* and *ImageTag* properties from the `ContainerInventory` table by joining on `ContainerID` property.  You can include the *Name* property (as it previously appeared in the `ContainerLog` table) from `KubepodInventory` table's *ContaineName* field by joining on the *ContainerID* property.